tools/packaging/README.md

```diff
@@ -3,7 +3,6 @@
 * [Introduction](#introduction)
 * [Build using OBS](#build-using-obs)
 * [Build in a container](#build-in-a-container)
-* [Build a snap package](#build-a-snap-package)
```

Comment:
@fidencio I am not sure about this. I understand we decided to move away from OBS packages, but we still would want to keep snap packaging for debian/ubuntu.

cc @egernst @bergwolf @jodh-intel @devimc

Reply:
I've updated the series dropping the patch that would drop the snap support, my bad for getting to eager and deleting that as well. :-)